spec/jobs/prepare_move_notifications_job_spec.rb

```diff
@@ -8,7 +8,7 @@
   let(:subscription) { create :subscription }
   let(:supplier) { create :supplier, name: 'test', subscriptions: [subscription] }
   let(:location) { create :location, suppliers: [supplier] }
-  let(:move) { create :move, from_location: location }
+  let(:move) { create :move, from_location: location, supplier: supplier }
```

Comment:
We can checkout the master version of this file and the tests will still pass :)